lib/api/asset/v1/service.test.ts

```diff
@@ -74,6 +77,13 @@ describe("useAsset", () => {
         numberOfLivingRooms: 1,
         windowType: "DBL",
         yearConstructed: "2077",
+        numberOfSingleBeds: 1,
+        numberOfDoubleBeds: 1,
+        numberOfFloors: 2,
+        totalBlockFloors: 5,
+        heating: "gas",
+        propertyFactor: "factor",
+        architecturalType: "type",
```

Comment:
Perhaps it would be useful to use real data from the sheet here given other fields seem to do so :thinking:

Reply:
Replaced with sample of data from spreadsheet
```
heating: "FCB",
propertyFactor: "4.5",
architecturalType: "HIGH RISE FLAT",
```